src/main/resources/ca/corefacility/bioinformatics/irida/database/changesets/unreleased/refactor-client-details.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+
+    <changeSet id="refactor-client-details" author="eric">
+
+        <dropTable tableName="client_details_additional_information" />
+        <dropTable tableName="client_details_additional_information_AUD" />
+
+        <dropTable tableName="client_details_authorities" />
+        <dropTable tableName="client_details_authorities_AUD" />
+
+        <dropTable tableName="client_details_auto_approvable_scope" />
```

Comment:
I verified that `read` and `write` were stored separately and thus I did not need to migrate those existing values over. All the other fields could not be set through the UI or were even being set in the code, so that is why I dropped those tables without migrations. I manually verified this data on all of our IRIDA installs. So yes it will all just work as is for existing IRIDA instances.

Reply:
Okay, so long as it will all just work.